### ISSUE TITLE
♿ a11y(bal-nav): Meta nav links as nav with linklist

### ISSUE THREAD

**Issue by carolabes** [baloise/design-system#1414]
### Description of this issue

* Meta nav links are not recognized as group by screen reader
* single element is read like => private customers, tab 1 of 1
* correctly implemented on mobile devices

<img width="1654" alt="meta-nav-links" src="https://github.com/baloise/design-system/assets/172396366/85aac40c-0ed7-43f1-bf39-a4b16be897be">

### How to fix it

* Should be recognized as group, so the screenreader reads e.g. => private customers - Link - 1 of 4
* Should be implemented as nav with link list

### Additional Information

Level A
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by m4rc0z** [comment]
i had a short check on baloise.ch and if I select the first element of the tab group (Privatkunden) it gets read like (Windows Narrator Screen Reader):

Privatkunden - register element 1 out of 4 selected

**Comment by carolabes** [comment]
I checked Baloise.ch on my windows device and every element is read single. There is no group. Tested with NVDA and Windows Build-in Screenreader. Windows build-in screenreader reads "register 0 elements" at the beginning.

Please check again

**Comment by hirsch88** [comment]
solved with v17.3.0
